### ISSUE TITLE
False positive SlevomatCodingStandard.Exceptions.RequireNonCapturingCatch.NonCapturingCatchRequired

### ISSUE THREAD

**Issue by Chi-teck** [slevomat/coding-standard#1647]
Version: slevomat/coding-standard 8.14.1

Code to reproduce:
```php
try {
  // Do something.
}
catch (\Exception $exception) {
  // Intentionally empty.
}
finally {
  \var_dump(isset($exception));
}
```

Error:
Non-capturing catch is required.

The code example looks weird but it is valid.  Apparently the corresponding sniff does not check usage of `$exception` variable outside of `catch` block.

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/commit/d82a6713c0063ba31e1d2bbc8a6df22798a2e370
